docs/containers-policy.json.5.md

```diff
@@ -110,6 +110,12 @@ The `tarball:` transport refers to tarred up container root filesystems.
 
 Scopes are ignored.
 
+### `containers-storage:`
+
+The `containers-storage` transport refers to the image location in a local containers storage.
+
+Supported scopes are for a specific image(s) or `""` for all images located in the containers-storage.
```

Comment:
The supported scopes are actually quite different (notably they always start with a `[…]` store specification), see `storageReference.PolicyConfiguration{Identity,Namespaces}`.